frontend/src/pages/create_campaign/Roles/CreateRoleForm.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { IconButton } from "@mui/material";
+import AddIcon from "@mui/icons-material/Add";
+import {
+  CreateRoleFormControl,
+  CreateRoleFormGroup,
+  CreateRoleQuantity,
+  CreateRoleName,
+} from "./rolesTab.styled";
+
+const CreateRoleForm = ({
+  newRoleQty,
+  setNewRoleQty,
+  newRoleName,
+  setNewRoleName,
+  addRole,
+}) => (
+  <CreateRoleFormControl>
```

Comment:
pet peeve but since its all inside the CreateRoleForm.js file, you dont need to prefix it all with CreateRole

FormControl, FromGroup, Quantity are all fine names

Reply:
FormControl and FormGroup are the names of the mui components themselves, is it okay to shadow the default names or should I pick something else so that they're distinct?